src/components/Recipe.js

```diff
@@ -8,32 +8,19 @@ export default function Recipe(props) {
 
   // TODO (issue 26): Card/Modal image
   return (
-    <div className="container">
-      <div className="rounded-border d-flex flex-row align-items-center bg-light">
-        <Card border="dark">
+    <>
+      <Card border="dark" bg="light" className="d-flex rounded-border">
+        <button className="card-title" onClick={() => setModal(true)}>
           <Card.Img variant="top" src="" />
-          <Card.Title>
-            <button className="card-title" onClick={() => setModal(true)}>
-              <strong>{name}</strong>
-            </button>
-          </Card.Title>
-          <Card.Body>
-            <div>Time: {time} min</div>
-            <div>Created by: {createdBy}</div>
-          </Card.Body>
-        </Card>
-      </div>
-      <Modal
-        show={modal}
-        onHide={() => setModal(false)}
-        centered="true"
-        size="lg"
-      >
+          <Card.Title>{name}</Card.Title>
+        </button>
+      </Card>
+      <Modal show={modal} onHide={() => setModal(false)} centered size="lg">
         <Modal.Header closeButton>
           <Modal.Title>{name}</Modal.Title>
           <Modal.Title className="modal-time">{time}</Modal.Title>
         </Modal.Header>
-        <img src="" alt={name} />
+        <Image src="" alt={name} fluid/>
```

Comment:
why Image instead of img?

Also, recommend adding a max width/height so the image doesn't get too large

Reply:
added height attribute and changed element to img